### DISCUSSION TITLE
Render HTML entities from {locale}.json

### DISCUSSION THREAD

**Discussion by ghost** [amannn/next-intl#852] [Q&A] [answered]
Hi there!
First, thanks a lot for this awesome project :pray: 

I'm using this library with `next-14` and I'm trying to use HTML entities within the locale JSON file. To give more context:
```json
{
    "example": "lorem ipsum&nbsp;ispum"
}
```
and the tsx file where it is used: 
```tsx
/* Imports */

export default function ExamplePage() {
  const t = useTranslations();
  return <h1>{t("example")}</h1>
}
```
It seems that it's always escaped, and the character is considered as a string. This is rendered as follows on the web page:
```html
lorem ipsum&nbsp;ispum
<!-- Instead of -->
lorem ipsum ispum
```

To fix the problem, I tried to do it with either `rich` and `raw` but it keeps displaying it the same.

I already search in the issues and the discussions, but I couldn't find anything related to it. What am I doing wrong and would there be a way to achieve that?

**Comment by amannn** [comment, marked as answer]
You can solve this by using the actual character instead of the HTML entity. In case of a `&nbsp;`, this is ` ` (you can copy-paste this). Wikipedia typically has samples where these special characters can be copied from (e.g. [Non-breaking space](https://en.wikipedia.org/wiki/Non-breaking_space)).

Hope this helps!

**Reply by ghost** [reply]
It helps! Thanks a :pray:

**Reply by abhishekP-bacancy** [reply]
How do we render `<strong>ipsum</strong>` or `<b>ipsum</b>` tags?
